base/applications/magnify/magnifier.c

```diff
@@ -931,12 +931,9 @@ INT_PTR CALLBACK OptionsProc(HWND hDlg, UINT message, WPARAM wParam, LPARAM lPar
                 if (HIWORD(wParam) == CBN_SELCHANGE)
                 {
                     HWND hCombo = GetDlgItem(hDlg,IDC_ZOOM);
-                    LPCTSTR currentZoomValue = TEXT("");
```

Comment:
To be clear, the original bug was that this isn't a buffer with space to  hold the current zoom value. Everything else looks reasonable in the original code.
Not saying the _ttoi necessarily has to stay... but it's not the problem.